add bitnami compat package for flux-kustomize-controller

sanjayk0508 opened the pull request:
Added bitnami compat package for flux-kustomize-controller, following the [Dockerfile](https://github.com/bitnami/containers/blob/main/bitnami/fluxcd-kustomize-controller/1/debian-12/Dockerfile)

### Pre-review Checklist

<!--
This checklist is mostly useful as a reminder of small things that can easily be
forgotten – it is meant as a helpful tool rather than hoops to jump through.

At the moment of this PR you have the most information on what all the change
will affect, so please take the time to jot it down.

Put an `x` in all the items that apply, make notes next to any that haven't been
addressed, and remove any items that are not relevant to this PR.

-->

#### For new package PRs only
<!-- remove if unrelated -->
- [x] This PR is marked as fixing a pre-existing package request bug
  - [x] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams
<!-- remove if unrelated -->
- [ ] The upstream project actually supports multiple concurrent versions.
- [x] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images
<!-- remove if unrelated -->
When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs
<!-- remove if unrelated -->
- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs
<!-- remove if unrelated -->
- [ ] The `epoch` field is reset to 0

#### For PRs that add patches
<!-- remove if unrelated -->
- [ ] Patch source is documented

sanjayk0508 committed:
add bitnami-compat

Signed-off-by: Sanjay Kumar <sanjay.kumar@chainguard.dev>

Dentrax commented on flux-kustomize-controller.yaml:
```diff
@@ -47,6 +47,25 @@ pipeline:
+          cp LICENSE ${{targets.subpkgdir}}/opt/bitnami/licenses/LICENSE
+    test:
+      pipeline:
+        - runs: |
```
Can we have `test/daemon-check-output` test here? Example:
https://github.com/wolfi-dev/os/blob/45214d8b0fd5427711f02174c01a3bc1c175b4ce/repmgr-17.yaml#L184-L201

sanjayk0508 replied:
tried digging in and experimenting, but it seems we can't use `test/daemon-check-output` because of the flux binary. Please let me know if you think it's doable or possible.

sanjayk0508 committed:
extend test

Signed-off-by: Sanjay Kumar <sanjay.kumar@chainguard.dev>

smoser committed:
flux-kustomize-controller - slight improvements to test

the test is still failing because the metrics endpoint does not include
the expected test.  I don't know if it is the righti expected text or
not.

the curl does succeed and we see the otuput when run.

kranurag7 committed:
update tests for fluxcd-kustomize-controller

adding tests for fluxcd-kustomize-controller and this is similar tests
that we are now doing for other flux-controllers.

Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

kranurag7 committed:
update subpackage name

Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

Dentrax commented on flux-kustomize-controller.yaml:
```diff
@@ -50,6 +50,41 @@ pipeline:
+          with:
+            setup: |
+              kubectl apply -f https://github.com/fluxcd/flux2/releases/latest/download/install.yaml
+              kubectl wait --for=condition=Ready nodes --all
```
nit: kwok pipeline already does this

Dentrax reviewed (approved):
LGTM, nice tests!
